.codecov.yml

```diff
@@ -0,0 +1,32 @@
+comment: off
+coverage:
+  status:
+    project:
+      default: off
+    core:
+        flags: core
+        target: auto
+        threshold: 100
```

Comment:
Could you quickly explain what's the meaning of this settings?

Reply:
Of course!

https://docs.codecov.io/v4.3.0/docs/commit-status

- flags: Allow us to have separate status checks for `core` and `interop`
- target: auto means that the coverage to compare to will be taken from `master`
- threshold: Percentage the coverage is allowed to drop (100 here means that CodeCov will always post a `success`, I did not want to enforce any particular number here)

Reply:
Thanks! I tried and failed at finding the docs.